test/e2e/specs/onboarding/signup__free.ts

```diff
@@ -115,7 +116,9 @@ describe( DataHelper.createSuiteTitle( 'Signup: WordPress.com Free' ), function
 			await startSiteFlow.clickButton( 'View designs' );
 		} );
 
-		it( 'See design picker screen', async function () {
+		// Skipped instead of reworked.
+		// https://github.com/Automattic/wp-calypso/issues/64418
+		it.skip( 'See design picker screen', async function () {
```

Comment:
I've chosen to skip this step in `Signup: Free` for now while you rework that spec @youbrokesomething.

Reply:
I'll include updates for this after this gets merged and I rebase. Your changes LGTM